EventVisualisation/Workflow/src/EveWorkflowHelper.cxx

```diff
@@ -298,7 +298,33 @@ void EveWorkflowHelper::drawAOD(EveWorkflowHelper::AODFullTrack const& track, fl
 
   auto const tr = o2::track::TrackParCov(track.x(), track.alpha(), arraypar, covpar);
 
-  addTrackToEvent(tr, GID::ITSTPCTRDTOF, trackTime, 0., GID::ITSTPCTRD);
+  addTrackToEvent(tr, detectorMapToGID(track.detectorMap()), trackTime, 0.);
```

Comment:
The `addTrackToEvent` takes GlobalTrackID (track type a.k.a Source + index in the source container + optional 2 bits) as an argument, while detectorMapToGID returns just a Source. Since here you don't care about the index, you can rename the detectorMapToGID to detectorMapToGIDSource and use it as 
```
addTrackToEvent(tr, GID{0, detectorMapToGIDSrc(track.detectorMap())}, trackTime, 0.);
```

Reply:
Thanks, this is now fixed.